scout/server/blueprints/variants/controllers.py

```diff
@@ -591,12 +591,38 @@ def upload_panel(store, institute_id, case_name, stream):
     return hgnc_symbols
 
 
+def gene_panel_choices(store, institute_obj, case_obj):
```

Comment:
This is the main function that populates the choices of the gene panel select for the variants filters. It is called every time you land on a variantS page. I was thinking that how it is now it's quite slim and fast. @dnil I like your suggestion to include also panel version and number of genes (as in a previous version of this PR) but to get this info it means that for each loop of the second for (line 612) you need to query the database for the panel object, so you know the version and number of genes. First loop (line 607) is fine because we know version and genes of case-specific panels.

I suggest a compromise here: specify version of panel and n. genes for case-specific panels and write instead 
`panel name (latest)` for the institute-specific panels. So no query is required. What do you say?

Reply:
I think that is quite acceptable. It will actually be a slight improvement over - and a clarification or reminder of the current mode of operation.